appveyor.yml

```diff
@@ -0,0 +1,53 @@
+version: "{branch}-ci-{build}"
+os:
+- Visual Studio 2013
+
+platform:
+- Win32
+
+configuration:
+- Debug 
+#- Release
+
+on_finish:
+#- ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+
+environment:
+  matrix:
+    - BOOST_ROOT: 'C:\Libraries\boost'
+      BOOST_INCLUDEDIR: 'C:\Libraries\boost\boost'
+      BOOST_LIBRARYDIR: 'C:\Libraries\boost\lib32-msvc-12.0'
+      ruby_version: "193"
+    - BOOST_ROOT: 'C:\Libraries\boost_1_58_0'
+      BOOST_INCLUDEDIR: 'C:\Libraries\boost_1_58_0\boost'
+      BOOST_LIBRARYDIR: 'C:\Libraries\boost_1_58_0\lib32-msvc-12.0'
+      ruby_version: "200"
```

Comment:
Ok, I think ruby 2.0.0 is latest that works with cucumber on windows. Not sure why ;)